examples/simple-chatbot/client/javascript/src/app.js

```diff
@@ -16,19 +16,34 @@
  * - Browser with WebRTC support
  */
 
-import { RTVIClient, RTVIEvent } from '@pipecat-ai/client-js';
+import { PipecatClient, RTVIEvent } from '@pipecat-ai/client-js';
 import { DailyTransport } from '@pipecat-ai/daily-transport';
 
+function _generateRandomWeather() {
```

Comment:
i've expanded this sample to implement function calling that requires a back/forth with the server. In this case, I am bulding the common "give me the weather" function call, but implementing the logic for getting the weather on the client. This is not how you would normally implement this. Typically, you'd probably do that on the server too, but you might need to ask the client for its location so there could still be a back/forth. I left it this way to be very clear about requiring the client to be in the mix.